continuous_integration/flake8_diff.sh

```diff
@@ -39,4 +68,5 @@ echo '--------------------------------------------------------------------------
 
 # Conservative approach: diff without context so that code that was
 # not changed does not create failures
-git diff --unified=0 $COMMIT | flake8 --diff && echo -e "No problem detected by flake8\n"
+git diff --unified=0 $COMMIT | flake8 --diff --show-source
```

Comment:
> Ok, I see. Makes sense.

Not really, but it kind of works and I have better things to do that understand the intricacies of bash if you ask me.